src/Mentat.UI/Views/Student/Index.cshtml

```diff
@@ -37,5 +36,33 @@
     </div>
 }
 
+<div class="row top-buffer col-sm-12">
+    <button type="button">Add Flashcard</button>
+</div>
+<div class="row top-buffer col-sm-12">
+    <table class="table top-buffer">
+        <thead>
+            <tr>
+                <th align="center">List of Flashcards</th>
+                <th></th>
+            </tr>
+        </thead>
+        <tbody>
+            @for (var i = 1; i <= Model.NumberOfFlashCards; i++)
```

Comment:
Can you zero index when working with arrays?

Reply:
Hi, @jonnywwright
You can, and originally, I had, but I was having to do a lot of conversions back and forth between the 0-base and 1-base. I am showing the index to the user, right? So, I need to know what number they are seeing and/or want to change to and I have to know how that relates to the list itself. I even had a VM member that's sole purpose was for the conversion, and yet I still found bugs where I had forgotten to do the conversion in my js methods. So, it was easier and less bug-prone for them to be the same number (index). Instead of one of them being base 1 (user friendly) and the other being base 0.

Let me know if that doesn't clear up your question.
Thank ya!
- Rachel

Reply:
I don't really follow your reasoning since this index isn't displayed to the user. That said you could try this

```csharp
foreach(var card in model.AvailableCards)
{
    // maybe see if there is an alternate display method that doesn't rely on the Model. 
    // but same idea
    Html.DisplayFor(m => card.CardQuestion);
}

```

Reply:
Hi, @jonnywwright 
I am displaying the index to the user with SelectedCardIndex, which is the same value that is held in i.
This for loop is the second time I am iterating over the list in this view. Keeping the loop the same as the one above makes it easier to maintain.

Let me know if you have any more questions,
Rachel

Reply:
I didn't know about the other one as I was only focusing on code changes for the PR. Now I think they should both be changed. However, we are in disagreement and typically these are resolved by a senior team member. @dyohn what would you consider best practice?

Reply:
nvm, this pull request is into someone elses branch not develop. My mistake, not my place to review this.